### PR TITLE
Duel overview page

### PR DESCRIPTION
**Pull request by nathanaelhoun**
## Changes

- [x] Added duel overview page
- [x] Only the played or playable rounds are shown
- [x] Play button is only activated when the the player can play the round
- [x] Add a reversed avatar attribute 

image
image

### CODE REVIEW

**Review by FrancoisPog — APPROVED** [review]
¡ Wꓕꓨꓶ

File: client/src/pages/DuelOverview.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import AuthService from "../services/auth.service";
+
+import Avatar from "../components/Avatar";
+
+const UserBadge = ({ user, reversed }) => {
```
Comment (valentinperignon) [inline client/src/pages/DuelOverview.js:8]: Maybe you could add the proptypes? Now that we're using the right package ^^

File: client/src/pages/DuelOverview.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        colorBG={user?.avatar?.colorBG}
+        colorBody={user?.avatar?.colorBody}
+        reversed={reversed}
+      />
+      <span>{user?.pseudo ?? "Pseudonyme"}</span>
+    </div>
+  );
+};
+
+const ResultBricks = ({ user, answers }) => (
```
Comment (valentinperignon) [inline client/src/pages/DuelOverview.js:26]: Same comment

**Review by valentinperignon — APPROVED** [review]
LGTM, just one comment!

File: client/src/pages/DuelOverview.js (hunk 1 above)
Comment (nathanaelhoun) [reply client/src/pages/DuelOverview.js:8]: You are right!

**Pull request merged**